cluster-scope/base/core/namespaces/physrisk-resilience/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- namespace.yaml
+namespace: physrisk-resilience
+components:
+- ../../../../components/limitranges/default
+- ../../../../components/resourcequotas/small
```

Comment:
I might be wrong , but based on sandbox namespace physical risk may not fit to the small quota especially when it comes to pvc . I wonder if you want to create resourcequota.yaml  (use sandbox as an example in local directory removing limit on pvc (I am not sure what is theirs requirements) and using values from large for the cpu and memory

Reply:
This is autogenerated from https://github.com/os-climate/physrisk-ui/issues/43. The adjustments sound fine to me, but should we do them here, or get the namespace in place and then open another PR to adjust it?

Reply:
Not to hold this PR I will approve and adjustments will be made as separate PR